compair/tests/test_compair.py

```diff
@@ -178,6 +178,9 @@ def login(self, username, password="password"):
         yield rv
         self.client.delete('/api/logout', follow_redirects=True)
 
+    def logout(self):
```

Comment:
You don't need this, use is automatically logged out when the `with self.login` is closed

Reply:
This was added as one of the test cases explicitly check if logout will fail during impersonation

Reply:
Ok, I think we should still remove this and handle the logout test in `LoginAPITests`
- need to add a new logout test (for all user roles)
- need to test logout when impersonating